Formula/swift-format.rb

```diff
@@ -0,0 +1,20 @@
+class SwiftFormat < Formula
+  desc "Formatting technology for Swift source code"
+  homepage "https://github.com/apple/swift-format"
+  url "https://github.com/apple/swift-format.git", :revision => "fc5a3da1d5d03143d7a31b12514349e2bf1aba8f"
```

Comment:
This needs to point at a tagged release.

Reply:
Hmm, this repo doesn't have a tagged release - it's 'release' is the (frozen) branch 'swift-5.1-branch'. Would that mean this formula is not acceptable for Homebrew?

Reply:
No: https://docs.brew.sh/Acceptable-Formulae#stable-versions